src/api/levelstats.js

```diff
@@ -0,0 +1,179 @@
+import express from 'express';
+import * as _ from 'lodash';
+import moment from 'moment';
+import { getPerfTracker, aggregateTrackers } from 'utils/perf';
+import { Op } from 'sequelize';
+import sequelize from 'data/sequelize';
+import { LevelStats, LevelStatsUpdate, Level, Time } from '../data/models';
+import * as PlayStats from '../data/models/PlayStats';
+
+const router = express.Router();
+
+// checks the current state of the database and processes the next X
+// times, resulting in updates to levelStats and levelStatsUpdates.
+export const doNext = async limit => {
```

Comment:
don't really want doNext and doAll in this file as they are for batch updating only. I was going to put them in the src/data/models/LevelStatsUpdate but then we get some circular dependency stuff going on. Maybe make a new file in utils. Any opinions?

Reply:
other cron kind of stuff is in utils as well, so makes sense to put it there